README.md

```diff
@@ -1 +1,66 @@
-# LeaderBoard-
\ No newline at end of file
+![](https://img.shields.io/badge/Microverse-blueviolet)
+
+# LeaderBoard
+
+> API TBD
+
+## Built With
+
+- HTML, CSS and Javascript
+- Git and Github
+
+
+## Live Demo (if available)
+
+https://emmykage.github.io/
+
+
```

Comment:
**[REQUIRED CHANGES]**

- Please, kindly update or remove the live demo link as this links to your portfolio and not to the current project.